### ISSUE TITLE
Bower package

### ISSUE THREAD

**Issue by Svenskunganka** [GetmeUK/ContentTools#29]
Hi! Awesome work done here, really enjoy this!

I'm currently writing an Angular wrapper for this project, and I wonder if you could add this package to bower? Will simplify a lot of things as I currently have to include the build in my project and every time you create a new compatible build I have to replace the old build with the new build.

Cheers!

**Comment by anthonyjb** [comment]
Yes that makes a lot of sense and it's a popular request, I'm currently looking ant node/NPM and bower support.

**Comment by Svenskunganka** [comment]
Great, thanks a bunch! Front-end, especially for angular, is still mostly backed by Bower but NPM is is getting more and more front-end packages too, so I definitely think NPM support is a good choice as well.

I'll link the repo to the angular wrapper here once I get a stable version ready.

**Comment by bfintal** [comment]
+1 for Bower

**Comment by mdartic** [comment]
+1 for bower too

**Comment by kand** [comment]
+1

**Comment by Solinx** [comment]
+1 for NPM / browserify support

**Comment by anthonyjb** [comment]
Just a quick update on the timing of this request, it's next on my priorities list once the test spec for ContentTools is complete. My hope is to have npm and bower distributions ready at the latest by middle of next month for all libraries in the collection.

**Comment by Solinx** [comment]
Thanks!

**Comment by mydudechris** [comment]
+1 for NPM + Browserify

**Comment by tmikaeld** [comment]
Another "me too" request for bower support.

**Comment by ksugiarto** [comment]
Hi guys, not trying to spamming this issues, but yes definitely +1 for Bower support. Anw, thanks for the awesome work of ContentTools really love it :D

**Comment by anthonyjb** [comment]
@mydudechris @tmikaeld @ksugiarto thanks for the support guys :) - and your patience - I am slowly but surely getting through the test spec work then I'm on to this.

**Comment by carnevlu** [comment]
+1 for NPM + Browserify

**Comment by anthonyjb** [comment]
Thanks to @toastercup for pushing the bower spec file ContentTools and all other libraries within the package can now be installed using bower. I'm going to open a new issue specifically for NPM and Browserify support.
